apidoc/Titanium/Android/Notification.yml

```diff
@@ -154,6 +154,22 @@ methods:
         summary: Allows to enable the indeterminate mode.
         type: Boolean
 
+  - name: addAction
+    summary: Add an action button to the notification
+    since: "6.2.0"
+    parameters:
+      - name: icon
+        summary: Icon of the action button as URL or resource ID
+        type: [Number, String]
```

Comment:
I think the icons have to use a holo dark theme on Android 4.x and a light material theme on Android 5.0 and above. Which means developers need to provide versioned drawable resources (ie: "drawable-mdpi-v21" for material). Or perhaps those details should go into the notification guide?